source/reference/template-apps.txt

```diff
@@ -97,13 +97,13 @@ with services and front-end code. You can:
 
 2. Follow on-screen guides to customize your app.
 
-3. Use the {+cli-bin+} to download the code for your app so you can build the frontend
-   (if one is included) and interact with the App through the cli.
+3. Use the {+cli-bin+} to download the code for your app so you can build the front end
+   (if one is included) and interact with the app through the cli.
 
-   If you already have {+cli-bin+} set up, you can use the {+cli-bin+}
+   If you already have {+cli-bin+} set up, you can use the following {+cli-bin+}
    :ref:`pull command <realm-cli-pull>` to download the code. Otherwise,
    click the :guilabel:`Pull front-end code` button in the welcome
-   section and follow the instructions that pop up.
+   section and follow the instructions that display.
```

Comment:
>    section and follow the instructions that display.

might be worth specifying *where* the instructions display. ie:

- the instructions that display in the terminal.
- the instructions in the terminal.
- etc.

Reply:
tweaked styling to sub-bullets to (hopefully) make it more clear that this is referring to instructions that display in the UI